source/extensions/filters/common/expr/context.cc

```diff
@@ -157,9 +157,13 @@ absl::optional<CelValue> PeerWrapper::operator[](CelValue key) const {
   auto value = key.StringOrDie().value();
   if (value == Address) {
     if (local_) {
-      return CelValue::CreateString(info_.downstreamLocalAddress()->asStringView());
+      if (info_.downstreamLocalAddress() != nullptr) {
+        return CelValue::CreateString(info_.downstreamLocalAddress()->asStringView());
```

Comment:
This is fine, but the stream info header states both can never be null.

Reply:
Ah OK. Should be fine then now since `TestStreamInfo` sets both. Originally it caught a ubsan bug when it wasn't set. Changed, since the fuzzer shouldn't hit that case.